node/src/command.rs

```diff
@@ -75,9 +72,9 @@ impl SubstrateCli for RelayChainCli {
 
     fn description() -> String {
         format!(
-            "Educhain Collator \n\nThe command-line arguments provided first will be passed to \
-             the parachain node, while the arguments provided after -- will be passed to the \
-             relay chain node.\n\n{} <parachain-args> -- <relay-chain-args>",
+            "Parachain Collator Template\n\nThe command-line arguments provided first will be \
```

Comment:
Any reason why Educhain branding is being removed?

Reply:
Nope, I was mistaken when I took this from the newest version. Will rectify